File: template/python/base-test.py
"""
{{_name_}}
"""

import unittest


class {{_expr_:substitute('{{_input_:name}}', '\w\+', '\u\0', '')}}(unittest.TestCase):
    def setUp(self):
        pass

    def tearDown(self):
        pass

    def test_something(self):
        {{_cursor_}}

File: template/java/base-test.java
{{_lang_util_:package}}

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

/**
 * {{_name_}}
 */
public class {{_name_}} {
	@Test
	public void test{{_input_:func}}() {
		{{_cursor_}}
	}

	@BeforeEach
	public void setup() {
	}

	@AfterEach
	public void tearDown() {
	}
}
{{_filter_:test}}
